functions/pipeline/extensions.csproj

```diff
@@ -0,0 +1,11 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Automatically generated. I believe if we use anything beyond HttpTrigger and TimerTriggers functions, we need these extensions installed, otherwise the other functions, queueTrigger, in our case, won't work.

https://github.com/Microsoft/vscode-azurefunctions/issues/382#issuecomment-417769686